### ISSUE TITLE
API-43:  Epic for Delivering MicroProfile 3.0

### ISSUE THREAD

**Issue by kwsutter** [OpenLiberty/open-liberty#7927]
This epic will be used to cover the platform MicroProfile 3.0:
https://bigblue.aha.io/features/API-43

MicroProfile 3.0 is based on Java EE 8 technologies. The updated component features for this release are tracked via their own features and #epics.

```
Health Check 2.0, https://bigblue.aha.io/features/LG-75, https://github.com/OpenLiberty/open-liberty/issues/7415     
Metrics 2.0, https://bigblue.aha.io/features/LG-15, https://github.com/OpenLiberty/open-liberty/issues/7946
Rest Client 1.3, https://bigblue.aha.io/features/API-42,  https://github.com/OpenLiberty/open-liberty/issues/7583
```

**Comment by kwsutter** [comment]
Link to WAD:  https://ibm.box.com/s/kvkulqauy4i01o3lhg783jlw5rcuciun

**Comment by kwsutter** [comment]
ID Issue:  https://github.ibm.com/was-liberty/liberty-docs/issues/1438

**Comment by kwsutter** [comment]
Feature Test Summary Issue #7939

**Comment by kwsutter** [comment]
Per the previous MicroProfile Platform convenience features, there is no work for accessibility, translation, serviceability, and globalization.  Can I request the associated approval tags?  @marikaj123  @gscottj  @donbourne

**Comment by kwsutter** [comment]
Per my past discussions with @arthurdm and the discussion in the [previous issue created for MP 2.1](https://github.com/OpenLiberty/ci.docker/issues/44), the creation of the OL specific docker images should be automated based off the changes that will go in via the [associated Packaging issue](https://github.ibm.com/was-liberty/WS-CD-Open/issues/17500).

For the WebSphere Liberty specific images, I have created the following Issue for tracking purposes:  https://github.ibm.com/WASCloudPrivate/roadmap-cloudpaks/issues/620

**Comment by arthurdm** [comment]
Hi @kwsutter - the difference here is that we would need a new `microProfile3` tag, so this would be more work than previous `2.x` changes.

**Comment by kwsutter** [comment]
Yes, @arthurdm, agree.  I wasn't positive if a new tag would be required or not, so I put that as a question in the Issue I created.  Thanks for responding so quickly!

**Comment by gscottj** [comment]
This feature does not add any new or change any existing UI. No AVT is required. Accessibility focal approved.

**Comment by tevans78** [comment]
This Epic will need FAT and Translation approvals but it is possible that the code will have to move to GA before they are obtained... TBC

**Comment by skasund** [comment]
L2 has requested STE slides for this feature.  The STE template can be found at the link below: 

https://ibm.box.com/s/1an42g7zdgmaj84w7dft0indqfgi8ffm

Please upload the completed slides to the same STE Archive BOX folder.  Thanks!

**Comment by marikaj123** [comment]
@kwsutter - Does this feature need translation?

**Comment by kwsutter** [comment]
@marikaj123 Per my earlier comment (https://github.com/OpenLiberty/open-liberty/issues/7927#issuecomment-503323631), there are no translation requirements.  Thanks!

**Comment by kwsutter** [comment]
@skasund, we have not had to provide STE slides for the MicroProfile platform convenience feature in the past (1.x and 2.x releases).  What has changed with this 3.0 release of MicroProfile where you think this is now required?  The "meat" of the release is actually in the child features (Health Check, Metrics, and Rest Client).  All of those features have their own respective STE work efforts.  Can you clarify your request?  Thanks.

**Comment by kwsutter** [comment]
@kldeen or @helyarp, could I get the ID approval for this Epic?  We have the ID feature in place and it seems to be tagged with 19007...  Thanks!

**Comment by skasund** [comment]
@kwsutter  If the information will be covered in other features, then a separate STE is not needed.  I'll approve this feature without an STE.

**Comment by marikaj123** [comment]
@kwsutter - Was the feature code enabled and included into the Liberty 19007 GM build?

**Comment by tevans78** [comment]
Yes, it was all delivered. Only think missing from this epic is the FAT focal approval.

**Comment by dave-waddling** [comment]
**FAT Focal Approval comment:** Hi, now that LG-50 has gotten its FAT Focal Approval there are no blockers to approving this feature.

**Comment by kwsutter** [comment]
Thanks, @dave-waddling!  All done now.
